blog/docs/articles/demystifying-activator-on-path.md

```diff
@@ -0,0 +1,182 @@
+# Demystifying Activator on path
+
+**Author: [Stavros Kontopoulos](https://twitter.com/s_kontopoulos), Principal Software Engineer @ RedHat**
+
+**Date: 2023-10-11**
+
+_In this blog post you will learn how to recognize when activator is on the data path and what it triggers that behavior._
+
+A knative service can operate in two modes:proxy mode and serve mode.
+When in proxy mode Activator is on the data path and it will stay on path until certain conditions are met.
+When these conditions are met Activator is removed from the path and the service transitions to serve mode.
+Although it was not always like that when a service scales from/to zero activator is added by default to the path.
+This default setting often confuses users for reasons we will see next as it is possible that activator will not
+be removed unless enough capacity is available. This is intended as one of the Activator's roles is to offer backpressure capabilities so that it can make sure that a service could serve the traffic.
```

Comment:
```suggestion
be removed unless enough capacity is available. This is intended as one of the Activator's roles is to offer backpressure capabilities so that a Knative service is not overloaded by incoming traffic.
```